### PR TITLE
History Descriptions

### PR DESCRIPTION
**Pull request by Waidhoferj**
Added descriptions and numbering to history tab

## Changes
Custom messages for the following actions
- Deleting a pill
- Assigning a pill
- Choosing a chart
- Applying a filter

Added numbers in chronological order where 1 is the earliest entry

## Test
- Perform actions that create custom messages in history and see results. 
- Verify that "Graph Changed" is not one of the messages. This is a fallback message and indicates that there is a missing description.

### CODE REVIEW

File: src/components/Sidebar/Tabs/DataTab.tsx [hunk 1]
```diff
@@ -155,7 +155,10 @@ export default function DataTab({
     if (clickedAxis) {
       updateClickedAxis('');
     }
-    saveSpecToHistory(newSpec);
+    saveSpecToHistory(
+      newSpec,
+      `Assigned the '${field}' field to ${activeEncoding}`
```
Comment (jahn96) [inline src/components/Sidebar/Tabs/DataTab.tsx:160]: Active Encoding is undefined

**Review by jahn96 — CHANGES_REQUESTED** [review]
Couldn't build the extension due to the error below

File: src/components/Sidebar/Tabs/DataTab.tsx (hunk 1 above)
Comment (Waidhoferj) [reply src/components/Sidebar/Tabs/DataTab.tsx]: Sorry about that

**Review by jahn96 — CHANGES_REQUESTED** [review]
Need to rebase main

File: bifrost_jupyter_extension.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "load_extensions": {
+    "bifrost_jupyter_extension/extension": true
+  }
+}
```
Comment (jahn96) [inline bifrost_jupyter_extension.json:5]: Sorry if this confuses you. I don't think we need `bifrost_jupyter_extension.json` and `bifrost.json` in our repo.

**Review by jahn96 — APPROVED** [review]
Other than this comment, all look good!

**Pull request merged**